src/DOM/WebStorage/Local.purs

```diff
@@ -0,0 +1,30 @@
+module DOM.WebStorage.Local
+( LocalStorage
+, getLocalStorage
+) where
+
+import DOM.WebStorage.Internal.Foreign
+
+import Control.Monad.Eff (Eff)
+import DOM (DOM)
+import Prelude ((<$>), (<<<))
+
+import DOM.WebStorage.String (class StrStorage)
+
+newtype LocalStorage = LocalStorage ForeignStorage
+
+instance strStorageLocalStorage :: StrStorage LocalStorage where
+  length = length <<< unLocalStorage
+  key = key <<< unLocalStorage
+  getItem = getItem <<< unLocalStorage
+  setItem = setItem <<< unLocalStorage
+  removeItem = removeItem <<< unLocalStorage
+  clear = clear <<< unLocalStorage
+
```

Comment:
Why is this necessary? Can't localStorage and sessionStorage just be of type ForeignStorage? What benefit has the distinction?